stable/ambassador/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.72.0
 description: A Helm chart for Datawire Ambassador
 name: ambassador
-version: 2.9.0
+version: 2.9.1
```

Comment:
new value field -> new feature -> minor. can you rebase and set it to 2.11.0? :)